components/user_settings/general/user_settings_general.tsx

```diff
@@ -255,7 +301,7 @@ class UserSettingsGeneralTab extends React.PureComponent {
         this.submitUser(user, true);
     }
 
-    submitUser = (user, emailUpdated) => {
+    submitUser = (user: object, emailUpdated: boolean | string) => {
```

Comment:
It looks like this is only be used as a `boolean`, if there's a call to this method passing a `string`, it should be cast to a `boolean`.